.github/workflows/release-suite-native-production.yml

```diff
@@ -125,7 +125,39 @@ jobs:
           BUILD_URL=$(eas build:view "$BUILD_ID" --json | jq -r '.artifacts.buildUrl')
           echo "build_url=$BUILD_URL" >> $GITHUB_OUTPUT
 
-      - name: Upload APK to S3
+      - name: Download APK
         run: |
           wget -O "Trezor-Suite-${{ steps.get_version.outputs.app_version }}.apk" "${{ steps.eas_build.outputs.build_url }}"
-          aws s3 cp "./Trezor-Suite-${{ steps.get_version.outputs.app_version }}.apk" s3://data.trezor.io/suite/releases/mobile/
+
+      - name: Upload APK as workflow artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: trezor-suite-apk
+          path: ./Trezor-Suite-${{ steps.get_version.outputs.app_version }}.apk
+          retention-days: 30
+
+      - name: Upload APK to S3
+        run: |
+          aws s3 cp "./Trezor-Suite-${{ steps.get_version.outputs.app_version }}.apk" s3://data.trezor.io/suite/releases/mobile/v${{ steps.get_version.outputs.app_version }}/
+
+      - name: Generate GitHub App token
+        id: trezor-bot-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.TREZOR_BOT_APP_ID }}
+          private-key: ${{ secrets.TREZOR_BOT_PRIVATE_KEY }}
+          owner: ${{ github.repository_owner }}
+          repositories: |
+            trezor/trezor-suite
+
+      - name: Prepare release notes
+        run: |
+          echo "https://data.trezor.io/suite/releases/mobile/v${{ steps.get_version.outputs.app_version }}/Trezor-Suite-${{ steps.get_version.outputs.app_version }}.apk" > release-notes.md
```

Comment:
Not final text, I guess... We can add there any copy.

Reply:
Why are we adding a URL to a file directly? I think having just a link to the data.treozor.io/suite/releases/mobile/ is sufficient. I thought that was the reason I'm adding the indexes there.